colin/cli/colin.py

```diff
@@ -10,7 +10,7 @@
 @click.command(context_settings=CONTEXT_SETTINGS)
 @click.argument('target', type=click.STRING)
 @click.option('--config', '-c', type=click.STRING,
-              help="Configuration name.")
+              help="Name of the configuration or path.")
```

Comment:
Why path? Path is also allowed? Do we really want it?

Reply:
Yes, that is exactly what we wanted..;-)

Reply:
You can then use something like this:
```
python3 -m colin.cli.colin -c ./config/fedora.json fedora:27
```